database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php

```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class AddNewxyfToPostsTable extends Migration
+class A extends Migration
```

Comment:
This is not needed

Reply:
I was having error with that so I changed the name but I deleted that migration already.

Reply:
database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php is shown as edited, not deleted.

Reply:
You'll need to use git get only the changes you need in this commit. I suggest to use a new branch to avoid to loose your work, Git can be tricky.